packages/esm-patient-chart-app/translations/en.json

```diff
@@ -28,22 +29,26 @@
   "confirmDeletingVisitTextWithStartAndEndDate": "Are you sure you want to delete {{visit}} which started {{visitStartDate}} and ended {{visitEndDate}}?",
   "confirmMarkAsAlive": "Are you sure, you want to mark patient as alive?",
   "confirmModifyingVisitDateToAccomodateEncounter": "The encounter date falls outside the designated visit date range. Would you like to modify the visit date to accommodate the new encounter date?",
+  "createdVisit": "Visit created",
+  "createVisit": "Create visit",
   "currentVisit": "Current Visit",
   "date": "Date",
   "dateAndTime": "Date & time",
+  "dateInFuture": "Date cannot be in future",
   "dateOfDeath": "Date of Death",
+  "datetimeOutOfRange": "Selected time is out of range",
   "delete": "Delete",
   "deleteEncounter": "Delete Encounter",
   "deleteEncounterConfirmationText": "Are you sure you want to delete this encounter? This action can't be undone.",
   "deleteThisEncounter": "Delete this encounter",
   "deleteVisit": "Delete visit",
-  "deleteVisitDialogHeader": "Are you sure you want to delete visit?",
+  "deleteVisitDialogHeader": "Are you sure you want to delete this visit?",
   "deletingVisit": "Deleting visit",
   "deletingVisitWillDeleteEncounters": "Deleting this visit will delete all associated encounters.",
   "diagnoses": "Diagnoses",
   "discard": "Discard",
   "dose": "Dose",
-  "editPastVisit": "Edit Past Visit",
+  "editPastVisit": "Edit past visit",
```

Comment:
There are a whole bunch of translation changes like this (i.e., shifts in capitalization). It would be nice to have an explanation for why these are part of this PR at all, especially because it's unclear why in some cases we are removing capitalizations ("Edit Past Visit" -> "Edit past visit") and in others we are adding them in ("Mark as alive" -> "Mark As Alive")